english/talk_to_neighbour/man/mulder.md

```diff
@@ -3,3 +3,4 @@ The man looked very stange. I was scared and looked frantically for the woman
 I saw the woman and ran toward her.
 
 [Talk to the woman](../woman/scully.md)
+[Drop kick the woman](../woman/attack.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.